test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
```

Comment:
I'm not sure I understand the description. This right now reads in the unit tests "morgan() tokens a custom" ??